crates/viewer/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -324,6 +324,40 @@ fn options_menu_ui(
         "Show 'Welcome screen' button",
     );
 
+    {
+        use re_renderer::video::DecodeHardwareAcceleration;
+
+        let hardware_acceleration = &mut app_options.video_decoder_hw_acceleration;
+
+        ui.horizontal(|ui| {
+            ui.label("Video Decoder:");
+            let response = egui::ComboBox::from_id_salt("video_decoder_hw_acceleration")
+                .selected_text(format!("{hardware_acceleration}"))
+                .show_ui(ui, |ui| {
+                    ui.selectable_value(
+                        hardware_acceleration,
+                        DecodeHardwareAcceleration::Auto,
+                        format!("{}", DecodeHardwareAcceleration::Auto),
+                    ) | ui.selectable_value(
+                        hardware_acceleration,
+                        DecodeHardwareAcceleration::PreferSoftware,
+                        format!("{}", DecodeHardwareAcceleration::PreferSoftware),
+                    ) | ui.selectable_value(
+                        hardware_acceleration,
+                        DecodeHardwareAcceleration::PreferHardware,
+                        format!("{}", DecodeHardwareAcceleration::PreferHardware),
+                    )
+                });
+
+            if response.inner.map_or(false, |inner| inner.changed()) {
+                // Video caches store video decoders that need to be recreated now with the right settings.
+                command_sender.send_system(SystemCommand::FlushViewerCacheType(
```

Comment:
Why isn't the `HardwareAcceleration` part of the cache key instead?

Reply:
hum. I don't know why this didn't occur to me. This code looked quite different when I first wrote it, changed after merging in Clement's changes. Could be that it was harder back then
This saves me quite a bit of trouble now ofc!